lib/bugsnag/integrations/railtie.rb

```diff
@@ -73,9 +74,16 @@ class Railtie < ::Rails::Railtie
     # @api private
     # @param event [Hash] details of the event to subscribe to
     def event_subscription(event)
-      ActiveSupport::Notifications.subscribe(event[:id]) do |*, data|
+      ActiveSupport::Notifications.subscribe(event[:id]) do |*, event_id, data|
         filtered_data = data.slice(*event[:allowed_data])
         filtered_data[:event_name] = event[:id]
+        filtered_data[:event_id] = event_id
+        if event[:id] == "sql.active_record"
+          binds = data[:binds].map { |bind| defined?(bind.name) ? [bind.name, '?'] : nil }
+          binds.select! { |bind| bind }
+          filtered_data[:bindings] = JSON.dump(binds.to_h) unless binds.empty?
+          filtered_data[:sql] = data[:sql].gsub(/(=|>|<|=>|=<|IN) (\S+)/, '\1 "?"')
```

Comment:
This gsub isn't working as I would expect for `IN` comparison (the 2nd element is still left in):
```
irb(main):001:0> puts "SELECT  `posts`.* FROM `posts` WHERE `posts`.`title` IN ('abc', 'def')".gsub(/(=|>|<|=>|=<|IN) (\S+)/, '\1 "?"')
SELECT  `posts`.* FROM `posts` WHERE `posts`.`title` IN "?" 'def')
```

Also, I'm sure there are other cases that aren't covered. See https://github.com/percona/percona-toolkit/blob/4c28e71092f1405cc75c5c4a1e28cfb9be94c570/bin/pt-fingerprint#L1635 to get an idea of how Percona Toolkit's pt-fingerprint approaches it (it isn't super simple).